common/kokkos-sampler/kp_sampler_skip.cpp

```diff
@@ -88,6 +100,11 @@ void kokkosp_init_library(const int loadSeq, const uint64_t interfaceVer,
   } else {
     tool_globFence = 0;
   }
+  if (NULL != tool_seed_str) {
+    tool_seed = atoi(tool_seed_str);
+  } else {
+    tool_seed = 1;
```

Comment:
Why? don't you check below for tool_seed < 0 to see if you want to set it via time?

Reply:
Fixed. Thanks.